BEIMA.Backend/Models/Requests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+
+namespace BEIMA.Backend.Models
+{
+    /// <summary>
+    /// Object representation of the data object in an add device request
+    /// </summary>
+    public class AddDeviceRequest
+    {
+        public string DeviceTag { get; set; }
+        public string DeviceTypeId { get; set; }
+        public string Manufacturer { get; set; }
+        public string ModelNum { get; set; }
+        public string SerialNum { get; set; }
+        public int YearManufactured { get; set; }
+        public string Notes { get; set; }
+        public Location Location { get; set; }
+        public Dictionary<string, string> Fields { get; set; } = new Dictionary<string, string>();
```

Comment:
Do we need images and documents in here as well?

Reply:
They will be attached to a different part of the mulitpart request. The models in the Requests.cs file will basically what the data section will serialize into.

`var req` incoming http request
`req.Files` contains all files
`req.Form['data']` contains one of the models in Requests.cs